keyboards/hazel/chaz/promicro/readme.md

```diff
@@ -0,0 +1,27 @@
+# Chaz
+
+![Chaz](https://i.imgur.com/FeZQFYAh.png)
+
+Low Profile, Choc Spaced QAZ-alike
+
+* Keyboard Maintainer: [Jason Hazel](https://github.com/jasonhazel)
+* Hardware Supported: Chaz, atmega32u4 ProMicro Compatible Controller
+* Hardware Availability: [Hazel's Garage](https://shop.hazel.cc/products/chaz), [Open Source](https://github.com/hazels-garage/chaz)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make hazel/chaz:default
+
+Flashing example for this keyboard:
+
+    make hazel/chaz:default:flash
```

Comment:
This file could just be moved to the top level keyboard directory, and the `keyboards/hazel/chaz/rp2040ce/readme.md` one removed.
```suggestion
* Hardware Supported: Chaz, atmega32u4 ProMicro Compatible Controller, RP2040 Community Edition Compatible Controller
* Hardware Availability: [Hazel's Garage](https://shop.hazel.cc/products/chaz), [Open Source](https://github.com/hazels-garage/chaz)

Make example for this keyboard (after setting up your build environment):

    make hazel/chaz:default
    make hazel/chaz/rp2040ce:default

Flashing example for this keyboard:

    make hazel/chaz:default:flash
    make hazel/chaz/rp2040ce:default:flash
```